Makefile

```diff
@@ -874,14 +886,36 @@ else
 	@$(MKDIR) $(@D)
 	@$(INSTALL) -m 0644 $< $@
 endif
-
+endif
 
 # --- Install-symlinks rules ---
 
 install-lib-symlinks: check-env $(MK_LIBS_SYML)
 
+ifeq ($(IS_WIN),yes)
+$(INSTALL_LIBDIR)/%.$(SHLIB_EXT): $(BASE_LIB_PATH)/%.$(SHLIB_EXT)
+ifeq ($(ENABLE_VERBOSE),yes)
+	@$(MKDIR) $(@D)
+	@$(INSTALL) -m 0644 $(BASE_LIB_PATH)/$(@F) $@
+else
+	@echo "Installing $(@F) into $(INSTALL_LIBDIR)/"
+	@$(MKDIR) $(@D)
+	@$(INSTALL) -m 0644 $(BASE_LIB_PATH)/$(@F) $@
+endif
+
+$(INSTALL_LIBDIR)/%.$(LIBBLIS_SO_MAJ_EXT): $(BASE_LIB_PATH)/%.$(LIBBLIS_SO_MAJ_EXT)
+ifeq ($(ENABLE_VERBOSE),yes)
+	@$(MKDIR) $(@D)
+	@$(INSTALL) -m 0644 $(BASE_LIB_PATH)/$(@F) $@
+else
+	@echo "Installing $(@F) into $(INSTALL_LIBDIR)/"
+	@$(MKDIR) $(@D)
+	@$(INSTALL) -m 0644 $(BASE_LIB_PATH)/$(@F) $@
+endif
+
+else
```

Comment:
So, it turns out this block is still bothering me. Since we don't install symlinks for Windows, the placement of these install rules can easily confuse the reader. Would it work to convert this `ifeq ($(IS_WIN),yes) ... else` block into an `else ... endif` block that is part of the `# Install shared library...` rule above (around line 880)? Here's how I envision the final result looking:
```make
# Install shared library containing .so major, minor, and build versions.
# Note: Installation rules for Windows does not include major, minor, and
# build version numbers.
ifeq ($(IS_WIN),no)

# Linux/OSX library (.so OR .dylib) installation rules.
$(INSTALL_LIBDIR)/%.$(LIBBLIS_SO_MMB_EXT): $(BASE_LIB_PATH)/%.$(SHLIB_EXT) $(CONFIG_MK_FILE)
ifeq ($(ENABLE_VERBOSE),yes)
    $(MKDIR) $(@D)
    $(INSTALL) -m 0644 $< $@
else
    @echo "Installing $(@F) into $(INSTALL_LIBDIR)/"
    @$(MKDIR) $(@D)
    @$(INSTALL) -m 0644 $< $@
endif

else

# Windows library (.dll and .lib) installation rules.
$(INSTALL_LIBDIR)/%.$(SHLIB_EXT): $(BASE_LIB_PATH)/%.$(SHLIB_EXT)
ifeq ($(ENABLE_VERBOSE),yes)
    @$(MKDIR) $(@D)
    @$(INSTALL) -m 0644 $(BASE_LIB_PATH)/$(@F) $@
else
    @echo "Installing $(@F) into $(INSTALL_LIBDIR)/"
    @$(MKDIR) $(@D)
    @$(INSTALL) -m 0644 $(BASE_LIB_PATH)/$(@F) $@
endif

$(INSTALL_LIBDIR)/%.$(LIBBLIS_SO_MAJ_EXT): $(BASE_LIB_PATH)/%.$(LIBBLIS_SO_MAJ_EXT)
ifeq ($(ENABLE_VERBOSE),yes)
    @$(MKDIR) $(@D)
    @$(INSTALL) -m 0644 $(BASE_LIB_PATH)/$(@F) $@
else
    @echo "Installing $(@F) into $(INSTALL_LIBDIR)/"
    @$(MKDIR) $(@D)
    @$(INSTALL) -m 0644 $(BASE_LIB_PATH)/$(@F) $@
endif

endif # ifeq ($(IS_WIN),no)

# --- Install-symlinks rules ---

install-lib-symlinks: check-env $(MK_LIBS_SYML)

# Note: Symlinks are not installed on Windows.
ifeq ($(IS_WIN),no)

# Install generic shared library symlink.
$(INSTALL_LIBDIR)/%.$(SHLIB_EXT): $(INSTALL_LIBDIR)/%.$(LIBBLIS_SO_MMB_EXT)
ifeq ($(ENABLE_VERBOSE),yes)
    $(SYMLINK) $(<F) $(@F)
    $(MV) $(@F) $(INSTALL_LIBDIR)/
else
    @echo "Installing symlink $(@F) into $(INSTALL_LIBDIR)/"
    @$(SYMLINK) $(<F) $(@F)
    @$(MV) $(@F) $(INSTALL_LIBDIR)/
endif

# Install shared library symlink containing only .so major version.
$(INSTALL_LIBDIR)/%.$(LIBBLIS_SO_MAJ_EXT): $(INSTALL_LIBDIR)/%.$(LIBBLIS_SO_MMB_EXT)
ifeq ($(ENABLE_VERBOSE),yes)
    $(SYMLINK) $(<F) $(@F)
    $(MV) $(@F) $(INSTALL_LIBDIR)/
else
    @echo "Installing symlink $(@F) into $(INSTALL_LIBDIR)/"
    @$(SYMLINK) $(<F) $(@F)
    @$(MV) $(@F) $(INSTALL_LIBDIR)/
endif

endif # ifeq ($(IS_WIN),no)
```